lighthouse-core/lib/asset-saver.js

```diff
@@ -255,22 +255,39 @@ async function saveAssets(artifacts, audits, pathWithBasename) {
  * Log trace(s) and associated devtoolsLog(s) to console.
  * @param {LH.Artifacts} artifacts
  * @param {LH.Audit.Results} audits
- * @return {Promise<void>}
+ * @return {Promise<Array<{name: string, contents: string}>>}
  */
-async function logAssets(artifacts, audits) {
```

Comment:
once we have the LR-side to process your nested artifacts, I think we can delete this logAssets function.

Reply:
done